src/helpers/volunteerHelper.ts

```diff
@@ -28,7 +28,16 @@ export const storeVolunteerAuthToken = (authToken?: string) => {
   }
 };
 
-export const volunteerAuthToken = () => SecureStore.getItemAsync(VOLUNTEER_AUTH_TOKEN);
+export const volunteerAuthToken = async () => {
```

Comment:
is the volunteer section a problem? why have you added code changes here? this has nothing todo with app start and splash screen behavior

Reply:
That's what I thought. But as far as I can see, it prevents the data in the application from loading.
This line of code causes a warning when the application is first launched. I updated the code to remove this warning.

Reply:
when we first start the app there is some warning about volunteer auth token?